### ISSUE TITLE
Autocomplete doesn't work on checkout when customer already has address

### ISSUE THREAD

**Issue by jmonschke** [shipperhq/module-address-autocomplete#5]
When you are adding a new address in checkout when the customer already has an address on file, the autocomplete functionality does not work.

**Comment by wsakaren** [comment]
thanks for raising we will investigate and get back to you. It will probably be next week as in the middle of an office move

**Comment by wsajosh** [comment]
Are you running the latest version of the module? I'm not able to reproduce the issue you're raising. I've attached a screen shot showing it working in that scenario on my local development machine.
![screen shot 2017-10-06 at 10 35 24](https://user-images.githubusercontent.com/1335244/31272257-3d6c15d0-aa82-11e7-86d8-0853816a056d.png)

**Comment by jmonschke** [comment]
The issue is that instead of putting your CSS in _module.less for googleplaces.css, its in a CSS file with a hard-coded z-index value. If you had instead put that in _module.less and had the following content, if there are any changes to the z-index values on the site, it would not cause issues.

```
@googleplaces__z-index: @modal__z-index + 1;

.pac-container {
    z-index: @googleplaces__z-index !important;
}

```

**Comment by wsakaren** [comment]
Has been escalated.

**Comment by wsakaren** [comment]
Internal JIRA - SHQ16-2328

**Comment by wsajosh** [comment]
This has been resolved in the latest release which is 20.1.0

**Comment by shimonah** [comment]
Issue is reproducible on 2.3.3 and 2.3.4 versions.
